qiskit_machine_learning/utils/loss_functions/loss_functions.py

```diff
@@ -20,37 +20,80 @@
 
 class Loss(ABC):
     """
-    Abstract base class for Loss.
+    Abstract base class for computing Loss.
+
     """
 
-    def __call__(self, predict, target):
+    def __call__(self, predict: np.ndarray, target: np.ndarray) -> float:
+        """
+        Args:
+            predict: a numpy array of predicted values using the model
+            target: a numpy array of the true values
+
+        Returns:
+            a float value of the loss function
+        """
         return self.evaluate(predict, target)
 
     @abstractmethod
-    def evaluate(self, predict, target):
-        """evaluate"""
+    def evaluate(self, predict: np.ndarray, target: np.ndarray):
+        """
+        An abstract method for evaluating the loss function
+
+        Args:
+            predict: a numpy array of predicted values using the model
+            target: a numpy array of the true values
+
+        Raises:
+            QiskitMachineLearningError: shapes of predict and target do not match
+        """
         raise NotImplementedError
 
     @abstractmethod
-    def gradient(self, predict, target):
-        """gradient"""
+    def gradient(self, predict: np.ndarray, target: np.ndarray):
```

Comment:
```suggestion
    def gradient(self, predict: np.ndarray, target: np.ndarray) -> float:
```
The typehint should show the return type

And likewise there should be a Returns: entry in the docstring that states the meaning of the return value

Reply:
updated